otel-manual-instrumentation/README.md

```diff
@@ -21,37 +21,20 @@ It consists of a basic shopping cart Python Flask web application with a backend
 
 ### Instructions
 
-1. Set up a local project directory:
-
-   ```sh
-   mkdir backend
-   cd backend
-   python3 -m venv venv
-   source ./venv/bin/activate
-   ```
-
-   See [Creating Virtual Environments](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) in the official Python documentation for more information.
+```sh
```

Comment:
Previously I was providing the steps without using this repo, but since now the steps is to use this repo, you need to add something about cloning or grabbing the `backend` folder from it, otherwise there is no mention of a `backend` folder until this point.